gradle.properties

```diff
@@ -1 +1 @@
-org.gradle.jvmargs=-Xmx2560M
```

Comment:
Can you undo this? This will make it fail to build on jenkins, should be better with 0.5 loom.